sympy/printing/tests/test_latex.py

```diff
@@ -149,6 +157,7 @@ def test_latex_symbols():
     assert latex(Symbol('omega') ** Symbol('beta')) == r"\omega^{\beta}"
 
 
+
```

Comment:
Here it is: https://github.com/sympy/sympy/wiki/Development-workflow#updating-your-pull-request
> Updating your pull request

>If you need to make changes to a pull request there is no need to close it. The best way to make a change is to add a new commit in you local repository and simply repeat push command:
```
>$ git commit
$ git push github 1234_sequences
```
> Note that if you do any rebasing or in any way edit your commit history, you will have to add the -f (force) option to the push command for it to work:
```
>$ git push -f github
```
>You don't need to do this if you merge, which is the recommended way.